### PR TITLE
Added the M365 Etiquette Bot to the page

### PR DESCRIPTION
**Pull request by kevmcdonk**
To help consumption of the content, I've added a Bot that uses QnA Maker to pull the contents from the Etiquette pages. Some more work needed on tidying the content

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -27,6 +27,12 @@ The tips are broken down in to specific services or approaches although this may
 - [Files](By-approach/files)
 - [Voice](By-approach/voice)
 
+## Using
+
+There are currently two ways to consume the content. One is to use the Github pages that you are on now. The other is to use the Bot below - the code for this is available at https://github.com/GreyHatBeard/m365-etiquette-bot. We will be working further on this so that you can use your own curated content for the Bot.
```
Comment (garrytrinder) [inline docs/index.md]: Should this be a markdown link?

**Review by garrytrinder — CHANGES_REQUESTED** [review]
All looks good @kevmcdonk except one minor comment, could you take a look?

File: docs/index.md (hunk 1 above)
Comment (kevmcdonk) [reply docs/index.md]: Good point! Changed

**Review by garrytrinder — APPROVED** [review]
All looks good! 

Approved 🚀

**Pull request merged**